README.md

```diff
@@ -242,6 +242,26 @@ Linux wheels are built in the [`manylinux` docker images](https://github.com/pyp
 - The project directory is mounted in the running Docker instance as `/project`, the output directory for the wheels as `/output`. In general, this is handled transparently by `cibuildwheel`. For a more finegrained level of control however, the root of the host file system is mounted as `/host`, allowing for example to access shared files, caches, etc. on the host file system.  Note that this is not available on CircleCI due to their Docker policies.
 - Alternative dockers images can be specified with the `CIBW_MANYLINUX_X86_64_IMAGE` and `CIBW_MANYLINUX_I686_IMAGE` options to allow for a custom, preconfigured build environment for the Linux builds. See [below](#options) for more details.
 
+### Modern C++ standards
```

Comment:
I'm a little bit in doubt on where we should add this, and in which amount of detail. I agree modern C++ versions are important, but `cibuildwheel`is independent of the language/compilation that generates/compiles the C extensions.

So in that case, it might be better to create a page with FAQ on different programming languages. And rather than explaining everything ourselves, we should link to the relevant resources so that users can find out more.